components/count-badge/README.md

```diff
@@ -24,6 +24,21 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
+| `text`, required | String | Descriptive text for the badge which will act as an accessible label and tooltip text when tooltips are enabled. |
+| `tab-stop`, default: `false` | Boolean | Optionally choose to make the badge a tab stop. |
+| `announce-changes`, default: `false` | Boolean | Optionally choose to announce changes to the badge with an aria-live region. If the number property is changed, the text will be read by screen-readers. |
+| `has-tooltip`, default: `false` | Boolean | Optionally choose to have a tooltip below the badge. |
+
+### Accessibility Properties
+
+To make your `d2l-count-badge` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `text`, required | Only the text will be read by screen-readers (not the number), so include the number in the text. |
+| `tab-stop` | A tab stop allows keyboard users to easily tab to the badge. Otherwise, keyboard users will need to arrow through to the badge. |

```

Comment:
```suggestion
| `tab-stop` | A tab stop allows screen-reader users to easily tab to the badge. Otherwise, screen-reader users will need to arrow through to the badge. |
```

Wonder if this is more accurate?  (Since just the tab-stop doesn't really add anything for keyboard users)